test-blink.cpp

```diff
@@ -32,19 +35,49 @@ int main(void)
         return 1;
     }
 
-    wiringPiSetupGpio();
-    pinMode(gpio, OUTPUT);
+    // Setup GPIO with libgpiod
+    struct gpiod_chip *chip;
+
+    chip = gpiod_chip_open("/dev/gpiochip0");
+    if (!chip)
+    {
+        std::cerr << "Failed to open gpiochip: /dev/gpiochip0" << std::endl;
+        return 1;
+    }
+
+    line = gpiod_chip_get_line(chip, gpio);
+    if (!line)
+    {
+        std::cerr << "Failed to get line! Output code: " << envGPIO << std::endl;
```

Comment:
Pretty hard to understand during runtime, especially if the user isn't familiar with the new GPIO terminology. 

```suggestion
        std::cerr << "Failed to get gpio line! Output code: " << envGPIO << std::endl;
```
To make the error more meaningful to the user.